changelogs/fragments/8794-Fixing-possible-concatination-error.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - community.general.proxmox inventory - Fixed a possible error on concatenating responses from proxmox. In case an API call unexpectedly returned an empty result, the inventory failed with a fatal error. Added check for empty response.
```

Comment:
```suggestion
  - proxmox inventory plugin - fixed a possible error on concatenating responses from proxmox. In case an API call unexpectedly returned an empty result, the inventory failed with a fatal error. Added check for empty response (https://github.com/ansible-collections/community.general/issues/8798, https://github.com/ansible-collections/community.general/pull/8794).
```